superduperdb/ext/llm/vllm.py

```diff
@@ -62,6 +63,47 @@ def build_post_data(self, prompt: str, **kwargs: dict[str, Any]) -> dict[str, An
         return {"prompt": prompt, **total_kwargs}
 
 
+class _VllmCore:
+    def __init__(self, **kwargs) -> None:
+        # Use kwargs to avoid incompatibility after vllm version upgrade
+        if "disable_log_stats" not in kwargs:
+            kwargs["disable_log_stats"] = True
```

Comment:
a log warning to make users aware of a kwarg override?

Reply:
This maintains the same behavior as the LLM log in the vLLM python sdk. Because AsyncLLMEngine adds two logs, it is not needed for actual use, so we disable them.